ci_framework/roles/libvirt_manager/tasks/create_network.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
this is kind of duplicating the network part in the create_layout.yml tasks file. Might be good to converge those changes. Care to iterate a bit on that?

Reply:
Yes... was a bit confused earlier

Made a split in `deploy_layout` so that I could use specific set of tasks.